config/init.go

```diff
@@ -110,6 +110,10 @@ const DefaultConnMgrGracePeriod = time.Second * 20
 // type.
 const DefaultConnMgrType = "basic"
 
+// DefaultResourceMgrMinInboundConns is a MAGIC number that probably a good
+// enough number of inbound conns to be a good network citizen.
+const DefaultResourceMgrMinInboundConns = 800
```

Comment:
We need basic regression test that ensures the implicit default autoscaling behavior is not changed silently.

MVP are two tests in existing `test/sharness/t0139-swarm-rcmgr.sh`:

1. set `ipfs config --json Swarm.ConnMgr.HighWater 1000`,  start node, read `ipfs swarm limit system | jq .ConnsInbound`, confirm it is 2x  `2000`

1. set `ipfs config --json Swarm.ConnMgr.HighWater 100`,  start node, read `ipfs swarm limit system | jq .ConnsInbound`, confirm it is > 800